### PR TITLE
Fixed prefixes on comments.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Regarding https://go.dev/wiki/CodeReviewComments#comment-sentences suggestion, comments should begin with the name of the thing being described and end in a period.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/maps/maps.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 // replacing this with `https://pkg.go.dev/golang.org/x/exp/maps` should be considered
 // when `x/exp/maps` graduates to stable.
+
```
Comment (trasc) [inline pkg/util/maps/maps.go:19]:
```suggestion
```

it's outdated

File: pkg/features/kube_features.go
Comment (trasc) [inline pkg/features/kube_features.go:1]: Let's not change this, to remain aligned with other k8s components (e.g. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go), maybe we can use some documentation generators on this.

File: pkg/util/maps/maps.go (hunk 1 above)
Comment (mbobrovskyi) [reply pkg/util/maps/maps.go]: Removed

File: pkg/features/kube_features.go
Comment (mbobrovskyi) [reply pkg/features/kube_features.go:1]: Reverted

**Review by trasc — COMMENTED** [review]
/lgtm
/approve

**Review by alculquicondor — COMMENTED** [review]
/approve

File: pkg/controller/admissionchecks/multikueue/workload.go [hunk 2]
```diff
@@ -71,13 +71,13 @@ type wlGroup struct {
...
-// the local wl is finished
+// IsFinished returns true if the local workload is finished.
 func (g *wlGroup) IsFinished() bool {
 	return apimeta.IsStatusConditionTrue(g.local.Status.Conditions, kueue.WorkloadFinished)
 }
 
-// returns true if there is a wl reserving quota
-// the string identifies the remote, ("" - local)
+// FirstReserving returns true if there is a workload reserving quota,
+// the string identifies the remote, ("" - local).
```
[the remark this answers is not in the document]
Comment (trasc) [reply pkg/controller/admissionchecks/multikueue/workload.go:80]: we can drop the `("" - local)` it is outdated.

In the early stages of multikueue, running on manager was also an option, and empty remote name wold have indicated that the workload should run locally.

[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply pkg/controller/admissionchecks/multikueue/workload.go]: Removed

**Pull request merged**